packages/vscode-docker-registries/src/wizard/showInputBox.ts

```
@@ -23,7 +23,7 @@ export async function showInputBox(options: RegistryWizardPromptStepOptions): Pr
                     resolve(inputBox.value);
                 }),
                 inputBox.onDidHide(() => {
-                    reject(new Error('Input box was hidden without providing any value.'));
+                    reject();
```

Should throw a cancellation error. VSCode has one built in.

---

and we catch this error in `prompt()` when we call `showInputBox()` right?

---

Nah, we can catch it higher up in the command's `callWithTelemetryAndErrorHandling` method in the Docker extension (when we move it)